Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,27 @@
+// storing the running totals, starting at 0
+var humanScore = 0;
+var computerScore = 0;
+
+// function human clicks rock, scores dont change
+function rock() {
+  $("rock").click;
```

Comment:
While you're on the right track in the sense that you want to grab the three buttons `$("#rock")`, `$("#paper")`, and `$("#scissors")` and hook it up to a click event which in turn is hooked up to a function, you can do that by putting the piece of HTML data (the button) and the event *inside* the function.

Try doing this instead, at the bottom of the file:

```
$("#rock").click(rock);
$("#paper").click(paper);
$("#scissors").click(scissors);
```

... and then remove lines 7, 15 and 23.